google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/collections/BasicTypeCollectionTest.java

```diff
@@ -58,13 +58,6 @@ protected Configuration constructAndConfigureConfiguration() {
 		return configuration;
 	}
 
-	@Before
-	public void cleanTables() {
```

Comment:
Why is this no longer necessary?

Reply:
This class now inherits from the modified `BaseCoreFunctionalTestCase` that we copied over which contains the cleanTables logic, so this is now redundant. The children classes no longer need clean table logic individually.